sql/policy_internal.sql

```diff
@@ -75,7 +80,7 @@ BEGIN
       INNER JOIN _timescaledb_catalog.chunk ch ON ch.table_name = pgc.relname AND ch.schema_name = pgns.nspname AND ch.hypertable_id = htid
     WHERE
       ch.dropped IS FALSE
-      AND (ch.status = 0 OR ch.status = 3)
+      AND ch.status IN (0,3,9,11)
```

Comment:
This is hard to decipher. Postgres has bitwise and for ints (`&`), maybe it would be more clear that way?

Reply:
Changed